### PR TITLE
Add generated manpages

### PR DESCRIPTION
**Pull request by besser82**
Add manpages for dnfdragora(1) and dnfdragora.yaml(5).  Manpages are generated using `sphinx-build`.

### CODE REVIEW

File: man/dnfdragora.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+``dnfdragora [options] [<args>...]``
+
+=============
+ Description
+=============
+
+.. _command_provides-label:
+
+DNFDragora a is a DNF frontend, based on rpmdragora from Mageia
```
Comment (Conan-Kudo) [inline man/dnfdragora.rst]: I get that this is generated, so where is this wrongly structured sentence coming from? It should be fixed.

Comment (besser82) [inline man/dnfdragora.rst]: It comes from in the README.md in the repo…  :P

File: man/dnfdragora_yaml.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+=========
+ Options
+=========
+
+``use_comps``
+    :ref:`boolean <boolean-label>`
+
+    If enabled group-sorting is read from from comps-file. This is recommended
+    for Fedora or other RedHat distributions currently. Default is false.
```
Comment (Conan-Kudo) [inline man/dnfdragora_yaml.rst]: `Fedora and other RedHat` -> `Fedora and other Red Hat-based`.

File: man/dnfdragora.rst (hunk 1 above)
Comment (Conan-Kudo) [inline man/dnfdragora.rst]: Ah, so you wrote this. Then basically take out the extra `a` after `DNFDragora`.

File: man/dnfdragora_yaml.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+``always_yes``
+    :ref:`boolean <boolean-label>`
+
+    If enabled dnfdragora will assume ``Yes`` where it would normally prompt
```
Comment (Conan-Kudo) [inline man/dnfdragora_yaml.rst]: Add comma after `If enabled`.

**Comment by besser82** [comment]
Should be fixed now.

**Comment by besser82** [comment]
Fixed.

File: man/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/conf.py.in ${CMAKE_CURRENT_BINARY_DIR}/conf.py @ONLY)
+
+add_custom_command(OUTPUT dnfdragora.1
```
Comment (Conan-Kudo) [inline man/CMakeLists.txt]: dnfdragora is a system administrative tool, so it should be in man page section 8.

**Comment by besser82** [comment]
Fixed…

**Comment by Conan-Kudo** [comment]
Merged, thanks!

**Pull request merged**